Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/nbclient/client.py", line 1314, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/jupyter_core/utils/__init__.py", line 165, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/asyncio/base_events.py", line 654, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/nbclient/client.py", line 709, in async_execute
    await self.async_execute_cell(
  File "/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/nbclient/client.py", line 1062, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/nbclient/client.py", line 918, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
# import basics
import matplotlib as mpl
import matplotlib.pyplot as plt
import numpy as np

# import pyathena 
import sys
sys.path.insert(0,'../../')
import pyathena as pa
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mModuleNotFoundError[0m                       Traceback (most recent call last)
Cell [0;32mIn[1], line 9[0m
[1;32m      7[0m [38;5;28;01mimport[39;00m [38;5;21;01msys[39;00m
[1;32m      8[0m sys[38;5;241m.[39mpath[38;5;241m.[39minsert([38;5;241m0[39m,[38;5;124m'[39m[38;5;124m../../[39m[38;5;124m'[39m)
[0;32m----> 9[0m [38;5;28;01mimport[39;00m [38;5;21;01mpyathena[39;00m [38;5;28;01mas[39;00m [38;5;21;01mpa[39;00m

File [0;32m~/work/pyathena/pyathena/docs/examples/../../pyathena/__init__.py:28[0m
[1;32m     25[0m [38;5;28;01mfrom[39;00m [38;5;21;01m.[39;00m[38;5;21;01mclassic[39;00m[38;5;21;01m.[39;00m[38;5;21;01mvtk_reader[39;00m [38;5;28;01mimport[39;00m AthenaDataSet [38;5;28;01mas[39;00m AthenaDataSetClassic
[1;32m     27[0m [38;5;66;03m# LoadSim classes[39;00m
[0;32m---> 28[0m [38;5;28;01mfrom[39;00m [38;5;21;01m.[39;00m[38;5;21;01mload_sim[39;00m [38;5;28;01mimport[39;00m LoadSim, LoadSimAll
[1;32m     30[0m [38;5;66;03m# Utils[39;00m
[1;32m     31[0m [38;5;28;01mfrom[39;00m [38;5;21;01m.[39;00m[38;5;21;01mutil[39;00m[38;5;21;01m.[39;00m[38;5;21;01munits[39;00m [38;5;28;01mimport[39;00m Units

File [0;32m~/work/pyathena/pyathena/docs/examples/../../pyathena/load_sim.py:19[0m
[1;32m     16[0m [38;5;28;01mimport[39;00m [38;5;21;01mxarray[39;00m [38;5;28;01mas[39;00m [38;5;21;01mxr[39;00m
[1;32m     17[0m [38;5;28;01mimport[39;00m [38;5;21;01mos[39;00m[38;5;21;01m.[39;00m[38;5;21;01mpath[39;00m [38;5;28;01mas[39;00m [38;5;21;01mosp[39;00m
[0;32m---> 19[0m [38;5;28;01mfrom[39;00m [38;5;21;01minherit_docstring[39;00m [38;5;28;01mimport[39;00m inherit_docstring
[1;32m     20[0m [38;5;28;01mfrom[39;00m [38;5;21;01mabc[39;00m [38;5;28;01mimport[39;00m ABC
[1;32m     21[0m [38;5;28;01mfrom[39;00m [38;5;21;01mcollections[39;00m[38;5;21;01m.[39;00m[38;5;21;01mabc[39;00m [38;5;28;01mimport[39;00m Mapping

[0;31mModuleNotFoundError[0m: No module named 'inherit_docstring'

